### PR TITLE
Aladdin generated examples for Vm

### PR DESCRIPTION
**Pull request by glGarg**
- Sort commands alphabetically in help files for easy diff.
 - Add Aladdin crafted examples.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -629,6 +659,10 @@
 long-summary: |
     For Linux VMs, disabling encryption is only permitted on data volumes.
     For Windows VMS, disabling encryption is permitted on both OS and data volumes.
+examples:
+  - name: Disable disk encryption on the OS disk and/or data disks. (autogenerated)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: This description is wrong. It is clearly disabling encryption only on data disks...

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 2]
```diff
@@ -857,6 +899,10 @@
 helps['vm image accept-terms'] = """
 type: command
 short-summary: Accept Azure Marketplace term so that the image can be used to create VMs
+examples:
+  - name: Accept Azure Marketplace term so that the image can be used to create VMs. (autogenerated)
+    text: az vm image accept-terms --urn publisher:offer:sku:version
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: `--urn publisher:offer:sku:version` this is not a real value.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py (hunk 1 above)
Comment (glGarg) [reply src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: I agree, we will keep this feedback in mind for getting better descriptions in future examples.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py (hunk 2 above)
Comment (glGarg) [reply src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Makes sense, we are currently in process of prioritize more concrete parameter values. As per offline discussion, we will still retain such values in the AI as they contain valuable information.

**Comment by JasonRShaver** [comment]
Ignore this:

Please react

**Pull request merged**